.github/workflows/build-fuzz-reusable.yml

```diff
@@ -0,0 +1,74 @@
+---
+name: Build - Fuzztests on L0 HW - Reusable
+
+on:
+  workflow_call:
+    inputs:
+      test_label:
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  fuzztest-build-hw:
+    name: Build and run fuzz tests on L0 HW
+    if: github.repository == 'oneapi-src/unified-runtime'  # run only on upstream; forks won't have the HW
+    strategy:
+      matrix:
+        build_type: [Debug, Release]
+        compiler: [{c: clang, cxx: clang++}]
+
+    runs-on: 'FUZZTESTS'
+
+    steps:
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+    - name: Install pip packages
+      run: pip install -r third_party/requirements.txt
+
+    - name: Download DPC++
+      run: |
+        wget -O ${{github.workspace}}/dpcpp_compiler.tar.gz https://github.com/intel/llvm/releases/download/nightly-2024-01-29/sycl_linux.tar.gz
+        mkdir dpcpp_compiler
+        tar -xvf ${{github.workspace}}/dpcpp_compiler.tar.gz -C dpcpp_compiler
+
+    - name: Build level zero with gcc
+      run: |
+        git clone -b v1.16.1 --depth=1 https://github.com/oneapi-src/level-zero.git ${{github.workspace}}/level-zero
+        cd ${{github.workspace}}/level-zero
+        cmake -B build -DCMAKE_BUILD_TYPE=${{matrix.build_type}} -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++
+        cmake --build build -j $(nproc)
+
+    - name: Configure CMake
+      # In order to use sanitizers, vm.mmap_rnd_bits=28 must be set in the system,
```

Comment:
this is not related to cmakes configure step. Move this to next to runs-on.

Reply:
Done